src/Akka.Monitoring.StatsD/ActorStatsDMonitor.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 using Akka.Monitoring.Impl;
-using StatsdClient;
+using JustEat.StatsD;
```

Comment:
I've changed StatsD client, as previous one was not updated for over 2 years. This one seems to be active and is also using the benchmarks and latests System.Memory stuff.